src/scripts/process_us_ca.py

```
@@ -0,0 +1,198 @@
+from datasets import load_dataset
+
+# ================= 설정 =================
+TARGET_PER_COUNTRY = 500
+OUTPUT_FILE = 'laws_data.json'
+
+COUNTRY_MAP = {'US': 1, 'CA': 2}
+
+# ================= 로직 =================
+
+def clean_text(text):
+    if not text: return ""
```

Ruff (0.14.7)

Multiple statements on one line (colon)

(E701)